spec/controllers/game_controller_spec.rb

```diff
@@ -0,0 +1,55 @@
+RSpec.describe GameController do
+  def app
+    Rack::Builder.parse_file('config.ru').first
+  end
+
+  let(:code_out_of_range) { '8888' }
+  let(:long_code) { '11111' }
+  let(:wrong_code) { '1111' }
```

Comment:
Are you sure that this code will be wrong all the time?

Reply:
correct code is being added explicitly in each of shared contexts

Reply:
so yes